### DISCUSSION TITLE
How to change the color of the slider?

### DISCUSSION THREAD

**Discussion by mattiasJohnson** [marimo-team/marimo#2140] [Q&A] [answered]
Would like to change it from dark blue to some other color. I have seen that you add css with the `.style()` method but I do not know what the name of that css property is?

It would also be useful to know the general approach on how to see the current styling of an element so I can change it easier.

**Comment by akshayka** [comment, marked as answer]
This is not possible.

We have limited support for theming: https://docs.marimo.io/guides/theming.html
